test/navigation/IdentificationNavigatorSpec.scala

```diff
@@ -31,31 +31,32 @@ class IdentificationNavigatorSpec extends SpecBase with ScalaCheckPropertyChecks
       val navigator = new IdentificationNavigator(NormalMode)
 
       "when answers complete" - {
-        "must redirect to check your answers" in {
+        "must redirect to location of goods type page" in {
           forAll(arbitraryIdentificationAnswers(emptyUserAnswers)) {
             answers =>
               navigator
                 .nextPage(answers)
-                .mustBe(routes.CheckIdentificationAnswersController.onPageLoad(answers.mrn))
-          }
-        }
-      }
-    }
-
-    "in CheckMode" - {
-
-      val navigator = new IdentificationNavigator(CheckMode)
-
-      "when answers complete" - {
-        "must redirect to check your answers" in {
-          forAll(arbitraryIdentificationAnswers(emptyUserAnswers)) {
-            answers =>
-              navigator
-                .nextPage(answers)
-                .mustBe(routes.CheckIdentificationAnswersController.onPageLoad(answers.mrn))
+                .mustBe(controllers.locationOfGoods.routes.TypeOfLocationController.onPageLoad(answers.mrn, NormalMode))
           }
         }
       }
     }
+// TODO: Add final check your answers page (CTCP-702)
+
+//    "in CheckMode" - {
+//
+//      val navigator = new IdentificationNavigator(CheckMode)
+//
+//      "when answers complete" - {
+//        "must redirect to check your answers" in {
+//          forAll(arbitraryIdentificationAnswers(emptyUserAnswers)) {
```

Comment:
Not a huge deal because it's commented out but this will need to be `arbitraryArrivalAnswers(emptyUserAnswers)` when this test gets implemented

Reply:
In fact it may be better to uncomment it but ignore it. Whichever way we think makes us less likely to forget about it 😄